tasks/stats.js

```diff
@@ -1,6 +1,6 @@
 var path = require('path');
 var fs = require('fs');
-var spawn = require('child_process').spawn;
+var spawn = require('cross-spawn');
```

Comment:
Ha nice catch. That's for that `spawn('npm', ['ls', '--json', '--only', 'prod']);` statement below - which will be obsolete when we switch to `npm@5` as the new `package-lock.json` file will do that same as the current `dist/npm-ls.json`.